CMakeLists.txt

```diff
@@ -9,8 +9,22 @@ include(FindPkgConfig)
 include(GNUInstallDirs)
 
 pkg_check_modules(UDEV REQUIRED libudev)
-pkg_check_modules(ANDROID_HEADERS REQUIRED android-headers)
-pkg_check_modules(ANDROID_HARDWARE REQUIRED libhardware)
+
+option(LIBHYBRIS_SUPPORT "Enable libhybris support" ON)
+
+if (LIBHYBRIS_SUPPORT)
+  pkg_check_modules(ANDROID_HEADERS android-headers)
+  pkg_check_modules(ANDROID_HARDWARE libhardware)
+  if(ANDROID_HEADERS_FOUND AND ANDROID_HARDWARE_FOUND)
+    message(STATUS "Bulding with libhybris support")
+    set(LIBHYBRIS true)
```

Comment:
Should be named `HAVE_LIBHYBRIS`.

Also, I would prefer a tristate that hard-fails when enabled but not found. However, my preliminary research shows that it's not straightforward to do so on CMake.

Reply:
It's not cmake has no way of detecting if the var is set by default or not...

Reply:
Fixed

Reply:
I wonder if we should hard-fail in this case...